app/services/mastodon_tasks_service.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+class MastodonTasksService
+  def self.daily
+    log_start_task('feeds::clear')
+    Feeds.clear
+    log_end_task('feeds::clear')
+
+    log_start_task('media::clear')
+    Media.clear
+    log_end_task('media::clear')
+
+    log_start_task('users::clear')
+    Users.clear
+    log_end_task('users::clear')
+
+    log_start_task('push::refresh')
+    Push.refresh
+    log_end_task('push::refresh')
+  end
+
+  class Media
+    class << self
+      def clear
```

Comment:
Could these all just be class methods on the existing AR classes being used here, instead of inside this module?